File: solutions/0551-student-attendance-record-i/solution.py
class Solution(object):
    def checkRecord(self, s):
        """
        :type s: str
        :rtype: bool
        """
        a=0
        l=0
        for i in s:
            if i=='A':
                a+=1
                l=0
            elif i=='L':
                l+=1
            else:
                l=0
            if l==3:
                return False
        return True if a<2 else False

File: solutions/0551-student-attendance-record-i/solution.c
bool checkRecord(char* s) {
    int l=0,a=0;
    for(int i=0;s[i];i++)
    {
        if(s[i]=='A')
        {
            a+=1;
            l=0;
        }
        else if(s[i]=='L')
        {
            l+=1;
            if(l==3)
                return false;
        }
        else
            l=0;
    }
    return a<2?true:false;
}
